cmd/blockscopy/main.go

```diff
@@ -226,7 +226,10 @@ func copyBlocks(ctx context.Context, cfg config, logger log.Logger, m *metrics)
 
 				blockDuration := time.Millisecond * time.Duration(meta.MaxTime-meta.MinTime)
 				if blockDuration < cfg.minBlockDuration {
-					level.Debug(logger).Log("msg", "skipping block, block duration is smaller than minimum duration", "blockDuration", blockDuration, "minimumDuration", cfg.minBlockDuration)
+					minTime := time.Unix(0, meta.MinTime*int64(time.Millisecond)).UTC()
```

Comment:
In newer Go version (I believe 1.17 and later) there is `time.UnixMilli()` that can be used.

Reply:
Right, but this project requirement is go 1.16. I would keep the upgrade separated.